README.md

```diff
@@ -48,16 +69,16 @@ Alternative implementation using Vert.x framework.
 - Gradle
 
 **Setup and Run:**
-```bash
+```sh
 cd vert.x
 ./gradlew shadowJar
 java -jar build/libs/teapot-3.5.4-fat.jar
 ```
 
 ## License
 
-See LICENSE file.
+0BSD (BSD Zero Clause License)
 
 ## Image Credit
 
-Teapot image by Joni Ludlow on Unsplash.
\ No newline at end of file
+Teapot image by [Joni Ludlow](https://unsplash.com/@joni_ludlow) on [Unsplash](https://unsplash.com/photos/rqaSSf7N3rc). Licensed under the Unsplash License.
```

Comment:
The URL will redirect to `https://unsplash.com/photos/white-blue-and-red-ceramic-teapot-pouring-water-on-white-ceramic-teacup-rqaSSf7N3rc`. Use the destination URL instead to save one redirect for the reader.